models/ShoeboxExample.osm

```diff
@@ -0,0 +1,302 @@
+
+OS:Version,
+  {7e1bc7bb-88c5-4343-b880-7bd835e15d52}, !- Handle
+  3.5.1,                                  !- Version Identifier
+  rc2;                                    !- Prerelease Identifier
```

Comment:
Let's avoid commiting a model done with a rc.

Reply:
3.3.0 is the oldest this will work in without changes (the SizingPeriodDesignDay was changed at 3.3.0), though it'd be easy to redo this in 3.0.0 for eg

Reply:
Good points, thanks for catching that